code/modules/research/designs/medical_designs.dm

```diff
@@ -1070,13 +1070,25 @@
 	surgery = /datum/surgery/advanced/lobotomy
 	research_icon_state = "surgery_head"
 
+/datum/design/surgery/lobotomy/mechanic
+	name = "Wetware OS Destructive Defragmentation"
+	desc = "A destructive robotic defragmentation method which guarantees removal of almost all brain traumas, but might cause another permanent trauma in return."
```

Comment:
Do we have cyberbrains? It seems odd to write this as if you're performing an operation on a computer rather than a regular lobotomy on a flesh brain which requires that you open up a mechanical head first.

Reply:
Well, the framing is basically that brains and computers are weirdly interchangeable with regards to augmented people, cyborgs and androids are concerned. That's what an MMI basically is after all, and we can't be sure whether or not the organ in that head is organic or synthetic. That's why I added the 'Wetware' bit. It's to be a bit ambiguous about it. And it also is a bit more flavourful.